src/cancelContentPlay.js

```diff
@@ -37,6 +37,16 @@ const cancelContentPlay = function(player) {
       player.el_.style.display = 'block';
       placeholder.remove();
     });
+
+    // On iOS 10 on iPhone, playsinline is added when pinching to return from
+    // fullscreen. Detect fullscreen change, remove placeholder and show player.
+    player.on('fullscreenchange', function () {
+      if (player.el_.hasAttribute('playsinline') && placeholder) {
```

Comment:
@marguinbc For iphone 10 - Tested both scenarios as suggested above:
`Scenario 1: setup player with playsinline and ad plugin (ima3 or freewheel)` 👍 
- I dont see a black div : case1 - Preroll is playing 👍 
                                          case2 - Returning from fullscreen 👍 

`Scenario 2: setup player without playsinline`
-I see a black div when returning from fullscreen

Checked by exiting fullscreen in all the 3 ways: Pinch-to-close, Done or Fullscreen button